app/__init__.py

```diff
@@ -10,7 +10,13 @@
 
 load_dotenv()
 app = Flask(__name__)
-mydb = MySQLDatabase(os.getenv("MYSQL_DATABASE"), user=os.getenv("MYSQL_USER"), password=os.getenv("MYSQL_PASSWORD"), host=os.getenv("MYSQL_HOST"), port=3306)
+
+mydb = None
```

Comment:
Why set this to None first? The else statement should make sure it's always assigned, no?

Reply:
I thought it would be good to declare mydb outside of the if/else statement scope because other languages (other than Python) would not be able to access mydb if it was only declared in the if/else statement scope. But I can definitely remove it if you wish! Up to your personal preference:blush:

Reply:
Totally cool either way! I was just curious.